### PR TITLE
:bug: Fix bug with resource Id for raw file

### PR DESCRIPTION
**Pull request by jobot0**
Linked to #60

### CODE REVIEW

File: trustkit/src/main/java/com/datatheorem/android/trustkit/config/TrustKitConfigurationParser.java [hunk 1]
```diff
@@ -4,6 +4,8 @@
 import android.content.Context;
 import androidx.annotation.NonNull;
 import android.text.TextUtils;
+import android.util.Log;
```
Comment (JulienNevo) [inline trustkit/src/main/java/com/datatheorem/android/trustkit/config/TrustKitConfigurationParser.java]: Unused import.

Comment (jobot0) [reply trustkit/src/main/java/com/datatheorem/android/trustkit/config/TrustKitConfigurationParser.java]: Good catch will fix it probably tomorrow

Comment (jobot0) [reply trustkit/src/main/java/com/datatheorem/android/trustkit/config/TrustKitConfigurationParser.java]: Should be good :)

**Comment by nabla-c0d3** [comment]
Thanks!

**Pull request merged**